lambda/Makefile

```diff
@@ -0,0 +1,43 @@
+LOCAL_ENDPOINT_HOST    ?=localhost
+LOCAL_ENDPOINT         =--endpoint-url=http://${LOCAL_ENDPOINT_HOST}
+SNS_ENDPOINT           =${LOCAL_ENDPOINT}:4575
+ES_ENDPOINT            =${LOCAL_ENDPOINT}:4578
+LAMBDA_ENDPOINT        =${LOCAL_ENDPOINT}:4574
+PROJ_ES_DOMAIN         =records
+PROJ_SNS_TOPIC         =data-update
+PROJ_AWS_REGION        =us-east-1
+PROJ_AWS_ACCESS_KEY_ID =999999
+PROJ_AWS_SECRET_KEY    =999999
+PROJ_ENV_VARS          =AWS_REGION=${PROJ_AWS_REGION} AWS_ACCESS_KEY_ID=${PROJ_AWS_ACCESS_KEY_ID} AWS_SECRET_ACCESS_KEY=${PROJ_AWS_SECRET_KEY}
+
+default: package upload subscribe
+
+package:
+	GOOS=linux go build -o main
+	zip lambda.zip main
+
+upload:
+	$(eval FUNCTION=$(shell aws $(LAMBDA_ENDPOINT) lambda list-functions | jq '.Functions[0].FunctionName // ""'))
```

Comment:
Will this prevent us from uploading more than one function?

Reply:
For now, yes. If we want to add more functions, this can be adapted for sure.

Reply:
:okay: